### PR TITLE
Move Check button into the File menu

### PR DESCRIPTION
**Pull request by MaelAudren**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -61,14 +61,13 @@
...
-                            <li>
-                            <a href="#" id="save-button" class="pointer dropdown-toggle" title="Save current workflow.">
-                                <i class="glyphicon glyphicon-hdd"></i> Save </a>
-                            </li>
-                            <li>
-                                <a href="#" id="close-button" class="pointer dropdown-toggle" title="Close current workflow.">
-                                <i class="glyphicon glyphicon-folder-close"></i> Close</a>
-                            </li>
+			  <li><a id="validate-button" href="#" class="pointer"
+                           title="Check the validity of current workflow."><i
```
Comment (tobwiens) [inline app/index.html]: Move the <i one line down

File: app/index.html [hunk 2]
```diff
@@ -61,14 +61,13 @@
...
+                            class="glyphicon glyphicon-ok "></i> Check</a></li>
```
Comment (tobwiens) [inline app/index.html]: <i

File: app/index.html [hunk 3]
```diff
@@ -61,14 +61,13 @@
...
+                            <li><a href="#" id="save-button" class="pointer dropdown-toggle" title="Save current workflow.">
+                                <i class="glyphicon glyphicon-hdd"></i> Save </a></li>
+                            <li><a href="#" id="close-button" class="pointer dropdown-toggle" title="Close current workflow.">
+                                <i class="glyphicon glyphicon-folder-close"></i> Close</a></li>
                             <li><a id="import-button" href="" class="pointer"
                                    title="Import existing workflows (.xml files)."><i
                                     class="glyphicon glyphicon-import"></i> Import as current workflow</a></li>
```
Comment (tobwiens) [inline app/index.html]: <i

File: app/index.html [hunk 4]
```diff
@@ -61,14 +61,13 @@
...
-                            </li>
+			  <li><a id="validate-button" href="#" class="pointer"
+                           title="Check the validity of current workflow."><i
+                            class="glyphicon glyphicon-ok "></i> Check</a></li>
+                            <li><a href="#" id="save-button" class="pointer dropdown-toggle" title="Save current workflow.">
+                                <i class="glyphicon glyphicon-hdd"></i> Save </a></li>
+                            <li><a href="#" id="close-button" class="pointer dropdown-toggle" title="Close current workflow.">
+                                <i class="glyphicon glyphicon-folder-close"></i> Close</a></li>
                             <li><a id="import-button" href="" class="pointer"
                                    title="Import existing workflows (.xml files)."><i
```
Comment (tobwiens) [inline app/index.html]: Move the <i one line down

File: app/index.html [hunk 5]
```diff
@@ -104,9 +103,7 @@
                            title="Browse the Workflow Catalog."><i
```
Comment (tobwiens) [inline app/index.html]: Move the <i one line down

File: app/index.html [hunk 6]
```diff
@@ -104,9 +103,7 @@
...
                             class="glyphicon glyphicon-folder-open"></i> Catalog</a></li>
```
Comment (tobwiens) [inline app/index.html]: <i

File: app/index.html [hunk 7]
```diff
@@ -104,9 +103,7 @@
...
                     <li class="divider-vertical"></li>
-                    <li><a id="validate-button" href="#" class="pointer"
-                           title="Check the validity of current workflow."><i
-                            class="glyphicon glyphicon-ok "></i> Check</a></li>
+
                     <li><a id="submit-button" href="#" class="pointer"
                            title="Execute current workflow on the Scheduler you are connected to."><i
```
Comment (tobwiens) [inline app/index.html]: Move the <i one line down

File: app/index.html [hunk 8]
```diff
@@ -104,9 +103,7 @@
...
                             class="glyphicon glyphicon-share "></i> Execute</a></li>
```
Comment (tobwiens) [inline app/index.html]: <i

File: app/index.html [hunk 9]
```diff
@@ -88,28 +87,26 @@
...
                         <a href="#" data-toggle="dropdown" id="edit-dropdown" > Edit <b class="caret"></b></a>
                         <ul class="dropdown-menu">
-                            <li><a id="clear-button" href="#" class="pointer" title="Clear current workflow."><i
-                                    class="glyphicon glyphicon-repeat "></i> Clear</a></li>
-                            <li><a id="undo-button" href="#" class="pointer"><i
-                                    class="glyphicon glyphicon-arrow-left "></i> Undo</a></li>
-                            <li><a id="redo-button" href="#" class="pointer"><i
-                                    class="glyphicon glyphicon-arrow-right "></i> Redo</a></li>
+                          <li><a id="clear-button" href="#" class="pointer" title="Clear current workflow.">
+			      <i class="glyphicon glyphicon-repeat "></i> Clear</a></li>
```
Comment (marcocast) [inline app/index.html]: is it correct to have a space at the end?

File: app/index.html [hunk 10]
```diff
@@ -88,28 +87,26 @@
...
+                          <li><a id="undo-button" href="#" class="pointer">
+			      <i class="glyphicon glyphicon-arrow-left "></i> Undo</a></li>
```
Comment (marcocast) [inline app/index.html]: is it correct to have a space at the end?

File: app/index.html [hunk 11]
```diff
@@ -88,28 +87,26 @@
...
+                          <li><a id="redo-button" href="#" class="pointer">
+			      <i class="glyphicon glyphicon-arrow-right "></i> Redo</a></li>
```
Comment (marcocast) [inline app/index.html]: is it correct to have a space at the end?

File: app/index.html [hunk 12]
```diff
@@ -88,28 +87,26 @@
...
                     <li class="divider-vertical"></li>
-                    <li><a id="validate-button" href="#" class="pointer"
-                           title="Check the validity of current workflow."><i
-                            class="glyphicon glyphicon-ok "></i> Check</a></li>
+
                     <li><a id="submit-button" href="#" class="pointer"
-                           title="Execute current workflow on the Scheduler you are connected to."><i
-                            class="glyphicon glyphicon-share "></i> Execute</a></li>
+                           title="Execute current workflow on the Scheduler you are connected to.">
+			<i class="glyphicon glyphicon-share "></i> Execute</a></li>
```
Comment (marcocast) [inline app/index.html]: is it correct to have a space at the end?

**Pull request merged**